src/Headers.test.ts

```diff
@@ -186,6 +186,11 @@ describe('.get()', () => {
     expect(headers.get('Accept')).toBeNull()
     expect(headers.get('AcCePt')).toBeNull()
   })
+
+  it('return an empty string for an empty header value', () => {
+    const headers = new Headers({ 'Content-Type': '' })
+    expect(headers.get('Content-Type')).toEqual('')
```

Comment:
This test ensures that we don't regress in a fix that correctly returns an empty string value instead of a null when the header was set to an empty string (an unlikely thing, but still)